app/controllers/idv/review_controller.rb

```diff
@@ -1,5 +1,5 @@
 module Idv
-  class ReviewController < ApplicationController
+  class ReviewController < Idv::BaseController
```

Comment:
Technically it's not necessary to include the `Idv::` prefix here, but I though it might help make it clearer the scope of "base" behaviors we're expecting.